examples/basic-typescript/.prettierrc.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
Is this the prettier config we will be using for all our templates now / the new site?